src/lib/accounts.js

```diff
@@ -38,3 +38,18 @@ export function getAccountsByType (cozy, accountType, accountsIndex) {
     })
   }
 }
+
+export function getAllAccounts (cozy, accountsIndex) {
+  if (accountsIndex) {
```

Comment:
Exactly my point : with still this method :D
https://github.com/cozy/cozy-data-connect/blob/master/src/lib/konnectors.js#L29